app/templates/components/build-header.hbs

```diff
@@ -37,6 +37,12 @@
           {{/if}}
         </li>
       {{/if}}
+      <li>
+        <a class="commit-branch-url" href="{{urlGitHubBranch}}" title="Look at the branch on GitHub">
+          {{inline-svg 'images/stroke-icons/icon-github' class="icon"}}
+          <span class="label-align">Branch {{item.branch.name}}</span>
+        </a>
+      </li>
```

Comment:
I seem to have found a bug:

![image](https://cloud.githubusercontent.com/assets/43280/20346858/b4e8d950-abfe-11e6-8d9e-526a59aa5c98.png)

While looking at this via PR deployment, it was working. Then I clicked the branch name to view it on GitHub, pressed back, and the branch name was missing. 😢

The link ends with `undefined` too, sadly.

(This isn’t really a comment on the template code, I just don’t think I can make a general comment without submitting the entire review.)